classes/local/store/object_file_system.php

```diff
@@ -334,7 +340,7 @@ public function delete_empty_folders($rootpath = null) {
             return 0;
         }
         $deletedircount = 0;
-        $iterator = new RecursiveDirectoryIterator($rootpath);
+        $iterator = new RecursiveDirectoryIterator($rootpath, RecursiveDirectoryIterator::FOLLOW_SYMLINKS);
```

Comment:
what is this about? there should not be any symlinks in filedir

Reply:
just ccurious why goes to `/var/lib/sitedata/filedir/16/fb` instead of `/var/lib/sitedata/ecs_marist_moodle_stage/filedir`